packages/core/src/browser/style/quick-title-bar.css

```diff
@@ -0,0 +1,53 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-quick-title-container {
+    display: flex;
+    flex-direction: row;
+    font-size: 11px;
+    padding: 3px 3px;
+    justify-content: flex-start;
+    align-items: center;
+    background: var(--theia-layout-color4);
+}
+
+.theia-quick-title-left-bar {
+    flex: 1px;
+    text-align: left;
+}
+
+.theia-quick-title-right-bar {
+    flex: 1px;
+    text-align: right;
```

Comment:
buttons and test are still don't look aligned, try to use `flex` here as display, it should auto-align icons

Reply:
reworked the styles